### PR TITLE
updated to install correct version of jdev 12.2.1.0.0 for adf v16

### PR DESCRIPTION
**Pull request by subodh-hatkar**

### CODE REVIEW

File: 12c_jdk8/Dockerfile [hunk 1]
```diff
@@ -29,15 +27,14 @@ ENV PATH $PATH:$M2_HOME/bin:$JAVA_HOME/bin:$ORACLE_HOME/bin
...
-	useradd oracle && \
+    useradd oracle && \
     chown -R oracle:oracle /usr/oracle && \
-    su oracle -c "java -jar jdev_suite_122120.jar \
+    su oracle -c "java -jar jdev_suite_122100.jar \
     -silent -responseFile /usr/oracle/middleware/jDevResponseFile \
     -invPtrLoc /usr/oracle/middleware/oraInst.loc" && \
-    rm -rf jdev_suite_122120.jar \
-	jdev_suite_1221202.jar \
+    rm -rf jdev_suite_1221202.jar \
```
Comment (luismsousa) [inline 12c_jdk8/Dockerfile]: This rm line does not have the right name, needs to be changed to the right jar.

everything else LGTM

**Review by luismsousa — COMMENTED** [review]
hi Subodh, have a look at the above comment.

**Comment by subodh-hatkar** [comment]
added right name to remove the jar.

**Review by luismsousa — APPROVED** [review]
LGTM

**Pull request merged**